pkgs/by-name/bu/butterfly/package.nix

```diff
@@ -0,0 +1,86 @@
+{
+  lib,
+  fetchFromGitHub,
+  flutter,
+  stdenv,
+  fetchurl,
+}:
+let
+  pname = "butterfly";
+  version = "2.2.1";
+  src = fetchFromGitHub {
+    owner = "LinwoodDev";
+    repo = "Butterfly";
+    rev = "v${version}";
+    hash = "sha256-TV7C0v3y9G44Df/x1ST8D0c0QqNBuuhzPBMKUyf/iwo=";
+  };
+  pdfium-binaries = stdenv.mkDerivation rec {
+    pname = "pdfium-binaries";
+    version = "5200";
+    src = fetchurl {
+      url = "https://github.com/bblanchon/pdfium-binaries/releases/download/chromium/${version}/pdfium-linux-x64.tgz";
+      hash = "sha256-Dmjn8MdhmltKkUMfrVFtZlzFLww0HXAbIarXGmxtQdk=";
+    };
+    dontUnpack = true;
+    installPhase = ''
+      runHook preInstall
+      mkdir $out
+      tar xvf $src -C $out/
+      runHook postInstall
+    '';
```

Comment:
missing `meta` field, especially `sourceProvenance` and `platforms`. Also why is this not built from source?